test/test_prototype_transforms.py

```diff
@@ -1,61 +0,0 @@
-import pytest
```

Comment:
Same deal as before. The tests are only partially compatible with the new features in the datasets. Thus, we remove them here and can re-add them when the transforms API is more stable.